public/content/defi/index.md

```diff
@@ -169,7 +169,7 @@ If exchange B's supply dropped suddenly and the user wasn't able to buy enough t
 
 To be able to do the above example in the traditional finance world, you'd need an enormous amount of money. These money-making strategies are only accessible to those with existing wealth. Flash loans are an example of a future where having money is not necessarily a prerequisite for making money.
 
-<ButtonLink isSecondary href="https://aave.com/docs/concepts/flash-loans">
+<ButtonLink isSecondary href="https://aave.com/">
```

Comment:
<img width="1195" alt="image" src="https://github.com/user-attachments/assets/e6a5bdd8-6244-4b48-9819-4d6462bd0555" />

Why are we redirecting this to aave.com instead of the working article?

```suggestion
<ButtonLink isSecondary href="https://aave.com/docs/concepts/flash-loans">
```